package.json

```diff
@@ -45,7 +45,8 @@
     "approve-size-tests": "node ./scripts/approve-size-golden.js",
     "integration-tests": "bazel test --test_tag_filters=-view-engine-only --build_tests_only -- //integration/... -//integration/size-test/...",
     "integration-tests:view-engine": "bazel test --test_tag_filters=view-engine-only --build_tests_only -- //integration/... -//integration/size-test/...",
-    "integration-tests:size-test": "bazel test //integration/size-test/..."
+    "integration-tests:size-test": "bazel test //integration/size-test/...",
+    "check-mdc-tests": "ts-node --project scripts/tsconfig.json scripts/check-mdc-tests.ts"
```

Comment:
Is this something manually to be run, or can we wire it up in CI?

Reply:
The idea is to eventually run it on the CI, but I wanted to add all the missing tests first and it still needs a way to exclude some tests that are irrelevant for the MDC version.